cmd/trade.go

```diff
@@ -609,6 +609,9 @@ func runTradeCmd(options inputs) {
 		logger.Fatal(l, fmt.Errorf("could not generate metrics tracker: %s", e))
 	}
 
+	tradeMetricsHandler := plugins.MakeTradeMetricsHandler()
+	metricsTracker.RegisterHandler(tradeMetricsHandler)
```

Comment:
the registration should happen closer to where the other handlers are registered in `makeFillTracker`

Reply:
If we're pre-initializing the handler (as per the suggestion [here](https://github.com/stellar/kelp/pull/574#discussion_r518378523)), then I don't think we need the registration at all.